_posts/2019-07-15-jun-updates.md

```diff
@@ -0,0 +1,38 @@
+---
+title: June Updates -  New releases, continue to foster diversity and inclusion in tech
+author: Hong Phuc Dang 
+date: 2019-07-15
+banner:
+  image: lights.jpg
+---
+### This is a recap of open source activities and development at Zalando in the month of June
+
+## Project Highlights 
+
+- [Kopf - Kubernetes Operator Pythonic Framework](https://github.com/zalando-incubator/kopf) now supports built-in resources and can be used to write controllers of any kind (pods, namespaces, mixed), not only of custom resources. Check out the latest release for more details [https://github.com/zalando-incubator/kopf/releases](https://github.com/zalando-incubator/kopf/releases)
+
+- [Skipper](https://github.com/zalando/skipper) publishes new releases weekly. Some of the important features were implemented such as support to proxy Kubernetes API server and support Kubernetes externalName services from ingress. 
+
+- [Kubernetes Ingress Controller for AWS](https://github.com/zalando-incubator/kube-ingress-aws-controller ) added dualstack and ssl-policy support in its last release. The controller helps to configure AWS application load balancers according to Kubernetes Ingress resources. 
+
+## Foster diversity and inclusion in tech 
+
+Zalando hosted the launch event of Persian Women In Tech Berlin. This is the first event of  the new Berlin chapter of the international organization of [Persian Women in Tech](http://www.persianwomenintech.com/). [Shery Brauner](https://twitter.com/sherybrauner), spoke about her career path from Iran to Germany, daring to take risks and now leading an engineering team at Zalando. 
+
+![shery](/assets/img/shery1.jpeg)
+
+Learn more about Zalando's initiatives around diversity and inclusion topics here: [https://jobs.zalando.com/en/diversity](https://jobs.zalando.com/en/diversity) 
+ 
+## Zalando Around The World 
+Meet and connect with Zalando representatives at tech events around the world: 
+
+[OpenExpo Europe](https://openexpoeurope.com), Madrid, Jun 20: [Hong Phuc Dang](https://twitter.com/hpdang), InnerSource manager, shared how Zalando applies open source practices internally, tools and processes that we use to foster alignment and collaboration within the company. View [Hong's slides](https://github.com/zalando/public-presentations/blob/master/files/2019-06-20_Open_Source_Within_Corporate_Walls-OpenEXPO_Madrid%20.pdf) 
+
+[ContainerDays](https://www.containerdays.io/), Hamburg, Jun 24 - 26: [Henning Jacobs](https://twitter.com/try_except_) presented his well-known [Kubernetes Failure Stories](https://srcco.de/posts/kubernetes-failure-stories.html)
```

Comment:
Video is now online: https://www.youtube.com/watch?v=LpFApeaGv7A

Reply:
@lmontrieux @hjacobs added the videos

Reply:
:+1: